core/src/main/java/jenkins/model/GlobalConfigurationCategory.java

```diff
@@ -73,11 +71,28 @@ public String getDisplayName() {
     public static class Security extends GlobalConfigurationCategory {
         @Override
         public String getShortDescription() {
-            return Messages.GlobalSecurityConfiguration_Description();
+            return hudson.security.Messages.GlobalSecurityConfiguration_Description();
         }
 
         public String getDisplayName() {
             return hudson.security.Messages.GlobalSecurityConfiguration_DisplayName();
         }
     }
+
+    /**
+     * Global configuration of tool locations and installers.
+     *
+     * @since TODO
+     */
+    @Extension
+    public static class Tools extends GlobalConfigurationCategory {
```

Comment:
:ant: or in a `jenkins.tools` package. I am thinking from the perspective of hypothetically splitting tool support to its own library plugin—then we would not want stuff like this inside more general types.